### DISCUSSION TITLE
Help with energy meter

### DISCUSSION THREAD

**Discussion by vlacemeister** [make-all/tuya-local#1858] [Q&A]
Please can anyone help me with this?

I have the Zemismart Tuya WiFi 3 Phase Energy Meter (https://vi.aliexpress.com/item/1005005879995196.html).
When I try do add it to Tuya-Local  I get a different set of devices (that I don´t have) to choose from.
Tuya category: dlq.
ip: set    192.168.1.xxx
product_id: fygozcnralhwbauo
Localkey: confirmed from Tuya IOT
device id: also tried w/ uuid
protocol: only 3.4 won´t return error

![other-device](https://github.com/make-all/tuya-local/assets/144171904/23e583ea-8f42-41e0-8e48-d6d6f5143e53)

thank you

**Comment by make-all** [comment]
With log messages, someone might be able to help. But not with just a description like that.

**Reply by vlacemeister** [reply]
Thank a lot for your fast reply.
I don´t know much about Home Assistant or how to get the info from the log...
Would that be enough to figure it out?

```
`Logger: custom_components.tuya_local.config_flow
Source: custom_components/tuya_local/config_flow.py:97
integration: Tuya Local (documentation, issues)
First occurred: April 29, 2024 at 8:44:47 AM (4 occurrences)
Last logged: 8:31:22 AM

Device matches zigbee_zn373186_temphumid_sensor with quality of 33%. DPS: {"updated_at": 1714391070.2807972, "1": 127070, "2": 0, "9": 0, "11": false, "12": false, "13": 0}
Device matches zigbee_zn373186_temphumid_sensor with quality of 33%. DPS: {"updated_at": 1714391119.845564, "1": 127071, "2": 0, "9": 0, "11": false, "12": false, "13": 0}
Device matches diivoo_wt05 with quality of 100%. DPS: {"updated_at": 1714432198.9150124, "11": 100, "103": 0, "104": false, "105": false, "106": 0, "112": "idle", "113": "idle", "114": "OFF", "117": "OFF"}
Device matches zigbee_zn373186_temphumid_sensor with quality of 33%. DPS: {"updated_at": 1714476660.2736034, "1": 127766, "2": 0, "9": 0, "11": false, "12": false, "13": 0}

Logger: custom_components.tuya_local.config_flow
Source: custom_components/tuya_local/config_flow.py:103
integration: Tuya Local (documentation, issues)
First occurred: April 29, 2024 at 8:44:47 AM (4 occurrences)
Last logged: 8:31:22 AM

Report this to https://github.com/make-all/tuya-local/issues/`
```
I hope this is right info.

**Reply by make-all** [reply]
Yes, this is the right info, I will take a closer look later when I have time.

**Reply by make-all** [reply]
It appears that the breaker_id that was showing up in the original logs is not present in your log.  Since this is non-essential info, there should be no problem in marking this as optional in the config to allow it to match when it is missing.
